DOCS-6157-added-proxy-install (#13)
mulesoft/docs-service-mesh: modules/ROOT/pages/uninstall-service-mesh.adoc

> 

Reviewer: HIDDEN TEXT edit:

= Uninstall Anypoint Service Mesh    ***EDITED***
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords: service mesh, microservices, uninstalling

Author: Done! Thanks!

> You might have to uninstall or delete Anypoint Service Mesh, for example, to troubleshoot issues. If you uninstall Service Mesh with bindings or adapters created, or have applications with an injected Envoy, the application stops working. The command removes all the Kubernetes components associated with Service Mesh project.

Reviewer: ```suggestion
You might need to uninstall or delete Anypoint Service Mesh: for example, to troubleshoot issues. If you uninstall Anypoint Service Mesh and it includes created bindings or adapters, or have if you created an application with an injected Envoy, the application stops working. The command removes all the Kubernetes components associated with the Anypoint Service Mesh project.
```

Author: Done! Thanks!

> == Recommendations for uninstallation

Reviewer: You do not need this subhead, and it is in fact misleading, because the following text provides not only prerequisites but also the task itself.
```suggestion
```

Author: Done! Thanks!

> Before you uninstall Service Mesh:

Reviewer: ```suggestion
Before you uninstall Anypoint Service Mesh:
```

Reviewer: HIDDEN TEXT edit

. Delete all bindings.
. Delete all adapters.
. Remove the injection label on the namespaces with provisioned applications.
. Redeploy your applications.
To uninstall Anypoint Service Mesh:   ***EDITED***
Run the  following command from the installer download URI:
`./service-mesh uninstall && rm service-mesh`

Author: Done! Thanks!
